.gitignore

```diff
@@ -41,3 +41,7 @@ substrate.code-workspace
 target/
 *.scale
 justfile
+
+# Dont commit
+.zed
+.target.rust-analyzer
```

Comment:
Can maybe remove here and make it a global configuration applies to all local projects.